pyproject.toml

```diff
@@ -133,6 +150,7 @@ rank-bm25 = { version = "^0.2.2", optional = true }
 pytest = { version = "^7", optional = true }
 pytest-asyncio = { version = "^0.23.0", optional = true }
 mock = { version = "^5", optional = true}
+openbb = "^4.3.5"
```

Comment:
1. make it as optional
2. put it under correct category

Reply:
Hiiii @Wendong-Fan , thanks for your reminding. I will make it as optional and put it under tool category.